ffv1.md

```diff
@@ -704,17 +708,20 @@ log2_run[41]={                                                |
 };                                                            |
                                                               |
 if (run_count == 0 && run_mode == 1) {                        |
-    if (get_bits(1)) {                                        |
+    if ( get_bits(1) ) {                                      |
```

Comment:
spaces near parenthesis are added here as you indicated ("Space between command (if, for, while) and arguments") but I don't see the change for other parts ("if (x + run_count <= w)", "if (run_index)"... below and lot of others), or did I miss something?

Reply:
It seems inconsistent, but I followed a rule:
  * If the statement in brackets ends with a bracket, I've kept/added the space, so it's clearer to read.
  * If it does not end in brackets, there's no need for space for readability.

Reply:
Ho, sorry, right you wrote it.
On my side I think it looks weird, I would prefer one of them only (all with space or none with space), but nothing big and we can keep that as is. @michaelni any preference?

Reply:
the "mixed" style looks a bit odd to me too. Not that it really matters, its just spaces ...

Reply:
I actually took the "))"-avoiding from existing code-styles in the paper. Thought it was on purpose that it was already mixed therefore.

Anyways: I can normalize all to non-space. No problem.

Shall I?

Reply:
Okay, I've now checked and seen that it's actually just this single if() with extra spaces. Thought there were more cases like that... :see_no_evil:

So I've now changed this so everything should be conform now :smile:

Reply:
I may have missed something but I see that you removed space from get_ur_golomb() part but not the one I am commenting (below if (run_count == 0 && run_mode == 1)).
PS: small stuff, just spaces, but as I find nothing else... :).